From 1b6e32a69a0289294e27f6214c4d8d67d461fe47 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Tue, 14 May 2024 06:13:21 -0600
Subject: [PATCH] Bump Security String to 2024-05-05

Implemented:
============
CVE:            References:    Type:  Severity:  Updated AOSP versions:
CVE-2024-23708  A-293301736    EoP    High       12, 12L, 13, 14
CVE-2024-23709  A-317780080    ID     High       12, 12L, 13, 14

Previously Implemented:
=======================
CVE:            References:    Type:  Severity:  Updated AOSP versions:  Prior Change:
CVE-2024-0024   A-293602317    EoP    High       12, 12L, 13, 14         9a5232bafe61
CVE-2024-0025   A-279428283    EoP    High       12, 12L, 13, 14         50fa805b6403
CVE-2024-23705  A-293602970    EoP    High       12, 12L, 13, 14         f856be82ebd9

Not Implemented:
================
None

Not Applicable (platform source):
=================================
CVE:            References:    Type:  Severity:  Updated AOSP versions:
CVE-2024-0043   A-295549388    EoP    High       12, 12L, 13, 14
CVE-2024-23706  A-328068777    EoP    Critical   14
CVE-2024-23707  A-316891059    EoP    High       14

Change-Id: Ic9157d0361716182961e61a69e80fd7bd0db8182
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 7475c4f985..b9e5f3fe54 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -250,7 +250,7 @@ ifndef PLATFORM_SECURITY_PATCH
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-    PLATFORM_SECURITY_PATCH := 2024-04-05
+    PLATFORM_SECURITY_PATCH := 2024-05-05
 endif
 .KATI_READONLY := PLATFORM_SECURITY_PATCH
 
-- 
2.40.1

